### PR TITLE
Convert tab indentations to spaces in *.resx files 

### PR DESCRIPTION
**Pull request by iSazonov**
Prepare to pass meta test #3504.

First commit convert tab indentations to spaces in *.resx files.
Second commit add Newlines at EOF.

### CODE REVIEW

File: src/Microsoft.PowerShell.ConsoleHost/resources/ConsoleHostStrings.resx [hunk 1]
```diff
@@ -143,7 +143,7 @@
 Windows PowerShell transcript start
 Start time: {0:yyyyMMddHHmmss}
 Username  : {1}\{2} 
-Machine	  : {3} ({4}) 
+Machinen  : {3} ({4})
```
Comment (JamesWTruher) [inline src/Microsoft.PowerShell.ConsoleHost/resources/ConsoleHostStrings.resx]: >Machinen [](start = 0, length = 8)

extra 'n'?

Comment (iSazonov) [inline src/Microsoft.PowerShell.ConsoleHost/resources/ConsoleHostStrings.resx]: Fexed.

File: src/Microsoft.PowerShell.Commands.Management/resources/ClipboardResources.resx [hunk 2]
```diff
@@ -131,4 +131,4 @@
   <data name="InvalidHtmlCombine" xml:space="preserve">
     <value>Html can only be combined with Html Text format.</value>
   </data>
-</root>
\ No newline at end of file
+</root>
```
Comment (mirichmo) [inline src/Microsoft.PowerShell.Commands.Management/resources/ClipboardResources.resx]: Missing newline here

File: src/Microsoft.PowerShell.Commands.Utility/resources/CsvCommandStrings.resx [hunk 3]
```diff
@@ -133,4 +133,4 @@
   <data name="UseDefaultNameForUnspecifiedHeader" xml:space="preserve">
     <value>One or more headers were not specified. Default names starting with "H" have been used in place of any missing headers.</value>
   </data>
-</root>
\ No newline at end of file
+</root>
```
Comment (mirichmo) [inline src/Microsoft.PowerShell.Commands.Utility/resources/CsvCommandStrings.resx]: And here

File: src/Microsoft.PowerShell.ConsoleHost/resources/CommandLineParameterParserStrings.resx [hunk 4]
```diff
@@ -269,4 +269,4 @@ EXAMPLES
   <data name="MissingConfigurationNameArgument" xml:space="preserve">
     <value>Cannot process the command because -Configuration requires an argument that is a remote endpoint configuration name. Specify this argument and try again.</value>
   </data>
-</root>
\ No newline at end of file
+</root>
```
Comment (mirichmo) [inline src/Microsoft.PowerShell.ConsoleHost/resources/CommandLineParameterParserStrings.resx]: Here

File: src/Microsoft.PowerShell.ConsoleHost/resources/ConsoleHostStrings.resx [hunk 5]
```diff
@@ -188,4 +188,4 @@ The current session does not support debugging; execution will continue.
   <data name="SlowProfileLoadingMessage" xml:space="preserve">
     <value>Loading personal and system profiles took {0}ms.</value>
   </data>
-</root>
\ No newline at end of file
+</root>
```
Comment (mirichmo) [inline src/Microsoft.PowerShell.ConsoleHost/resources/ConsoleHostStrings.resx]: A guess: I think your script was fairly consistent with these. If it made a change in the file, it didn't add an extra newline at the end. I'm going to stop commenting on each individual one. Please make it consistent throughout if you are going to make this change.

**Review by mirichmo — CHANGES_REQUESTED** [review]

**Comment by mirichmo** [comment]
@lzybkr - I remember that you had some thoughts on this topic so I wanted to make sure you are aware of this PR.

**Comment by lzybkr** [comment]
Well, my thoughts are basically - style/formatting changes shouldn't be piecemeal - we should deal with all the files at once. And I'd prefer to see one commit per "kind" of change, say replace tabs, or fix file encoding, or add/remove newlines.

The newline at the end is an annoying issue - some tools expect a newline on the last line (like `wc` and `git`), whereas a human reader, a blank line at the end needlessly takes up space on the screen, so I naturally delete it.

So my point here is - unless we have a pre-commit hook, it's a battle we can't win.

**Comment by iSazonov** [comment]
I prepared Add Metatests #3504 It cannot be activated until all the files are cleaned up.
The same applies to pre-commit hooks - if today we active it each contributor would have to make many changes to the formatings in addition to his code changes.

I believe we must first do the global file cleanup then activate the checks, first, for the current commits, then globally in the nightly tests.

We have already done a few steps:
@lzybkr  removed trailing whitespace #3001 (not in *.resx).
I converted tab indentations to spaces in *.cs files  #3551.

In the PR I convert tab indentations in *.resx. 
Next I plan to convert tab indentations *.test.ps1 (but I can add the commit to the PR if you want) and then remove trailing whitespace in *.resx.

And in the end I plan to add Newline at EOF in all files.

I did not follow the rule "all the files at once" because @TravisEz13 asked to do a work in smaller chunks to simplify a review.

Can we follow this roadmap?

**Comment by TravisEz13** [comment]
The risk of doing everything in one large change is that you won't be able to ever merge the change due to merge conflicts.

**Comment by iSazonov** [comment]
Conflict was resolved.
Newlines will be fixed in another PR.
Can we continue the code review?

**Comment by iSazonov** [comment]
@mirichmo Could we continue with the PR?

**Review by mirichmo — APPROVED** [review]

**Pull request merged**